reactMovieApp/src/components/MovieCard.js

```diff
@@ -11,4 +10,8 @@ function MovieCard({movies}){
 	);
```

Comment:
arrow func, inline return 

ESLint  doesn't complain about the file type extension?